cmd/icingadb/main.go

```diff
@@ -77,6 +79,8 @@ func run() int {
 		}
 	}
 
+	go monitorRedisSchema(logger, rc, checkRedisSchema(logger, rc, "0-0"))
```

Comment:
So I guess this extra call to `checkRedisSchema()` is done here to get in executed once before spawning the goroutine as it might itself exit with a fatal error. Maybe this could be made more explicit, just as the SQL schema check does here in main.